app/controllers/schools_controller.rb

```diff
@@ -116,10 +117,11 @@ def destroy
 
   def set_search_scope
     @tab = SchoolSearchComponent.sanitize_tab(search_params.fetch(:scope).to_sym)
+    @schools = current_user_admin? ? School.active : School.visible
```

Comment:
Would it be beneficial to put a notice for admins saying that this is happening?
'Note for admins: admins can also see schools marked as invisible.'